darwin/utils/utils.py

```diff
@@ -941,6 +945,21 @@ def _parse_annotators(annotators: List[Dict[str, Any]]) -> List[dt.AnnotationAut
     return [dt.AnnotationAuthor(annotator["full_name"], annotator["email"]) for annotator in annotators]
 
 
+def _parse_properties(properties: List[Dict[str, Any]]) -> Optional[List[dt.SelectedProperty]]:
```

Comment:
does dt.SelectedProperty exist? I don't see it in datatypes so shouldn't this point to darwin.future...SelectedProperty?

Reply:
You're right, the reason my testing worked because darwin.datatypes was already importing SelectedProperty from future..
Fixed in a new commit!